tests/k8s-azure/manifest/linux-vmss.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
still not right. the file has already been added in https://github.com/kubernetes/cloud-provider-azure/blob/master/tests/k8s-azure/manifest/linux-vmss.json.

Could you squash all the commits and rebase it to master branch?